build/languages.js

```diff
@@ -38,12 +38,14 @@ const languages = [
     { name: 'git_commit', identifiers: ['COMMIT_EDITMSG', 'MERGE_MSG'], source: 'text.git-commit' },
     { name: 'git_rebase', identifiers: ['git-rebase-todo'], source: 'text.git-rebase' },
     { name: 'go', language: 'go', identifiers: ['go', 'golang'], source: 'source.go' },
+    { name: 'graphql', language: 'graphql', identifiers: ['qgl', 'graphql'], source: 'source.graphql' },
```

Comment:
Is the identifier `qgl` intentional? I think `gql` is more common for the shorthand for graphql.